src/MassiveActionHandler.test.ts

```diff
@@ -150,11 +150,11 @@ describe('TestMassiveActionHandler', () => {
   it('returns a needToSeek block number if state already exists', async () => {
     const nextBlock = await actionReader.getNextBlock()
     await actionHandler.handleBlock(nextBlock, false)
-    expect(actionReader.currentBlockNumber).toBe(1)
+    expect(actionReader._currentBlockNumber).toBe(1)
 
     const nextBlock2 = await actionReader.getNextBlock()
     await actionHandler.handleBlock(nextBlock2, false)
-    expect(actionReader.currentBlockNumber).not.toBe(1)
+    expect(actionReader._currentBlockNumber).not.toBe(1)
```

Comment:
Shouldn’t this be a function call?

Reply:
Nope, it's a getter method, so you access it like a property.

Reply:
Ah reading helps 👍